content/en/docs/gke/deploy/project-setup.md

```diff
@@ -41,7 +41,8 @@ Follow these steps to set up your GCP project:
       iam.googleapis.com \
```

Comment:
Lines36-37:

- Please add a space separating the code block from the paragraph and then add bash to the right of the three backticks 👍as per @Bobgy 's comment